client/src/AppBundle/EventSubscriber/ReportSubmittedSubscriber.php

```diff
@@ -36,4 +49,17 @@ public function sendEmail(ReportSubmittedEvent $event)
             $this->mailer->sendReportSubmissionConfirmationEmail($event->getSubmittedBy(), $event->getSubmittedReport(), $newReport);
         }
     }
+
+    public function log(ReportSubmittedEvent $event)
```

Comment:
Sorry, had a last-minute thought to make this a bit clearer - we should probably make it clear to someone reading this subscriber that this logging only happens when a report is being resubmitted so something like `logResubmittedReport()`? Last comment, promise 😬

Reply:
Yeah no worries. Good point!